src/app/pharmacists/components/prescription-list/prescription-list.component.html

```diff
@@ -102,8 +102,7 @@ <h2 fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="10px">
           <!-- Expanded Content Column - The detail row is made up of this one column that spans across all columns -->
           <ng-container matColumnDef="expandedDetail">
             <td mat-cell *matCellDef="let element" [attr.colspan]="displayedColumns.length">
-              <div *ngIf="'status' in element" class="element-detail"
-                   [@detailExpand]="element._id == expandedElement?._id ? 'expanded' : 'collapsed'">
+              <div *ngIf="'status' in element" class="element-detail" [@detailExpand]="element._id == expandedElement?._id ? 'expanded' : 'collapsed'">
```

Comment:
La expresión  *ngIf=" ... in ..." no la esta tomando bien el navegador, lo marca como  error... modificar por *ngIf="element?.status"